### ISSUE TITLE
available libraries

### ISSUE THREAD

**Issue by silvau** [WilldooIT/Pentaho-reports-for-OpenERP#230]
I have a report who is  doing some replacements in a string field  and then prints the modified string in to the pdf report.

For replacement I'm using  a library called:  org.apache.commons.lang.StringUtils

When I run the report from Pentaho Report Designer. The report is doing it right.

When I put my prpt file in openerp .. I get the pdf  but the modified string is not shown.

When I ceck the catalina.out log from tomcat.. I have found:

"..Attempt to resolve method: replace() on undefined variable or class name: StringUtils "

This make me think that  libraries are not present. 

Is there a way to use libraries in reports to be used in openerp?. What is the right way to do it?. 

Is the .war file the problem?. How should the .war file be constructed to include needed libraries?

**Comment by richard-willdooit** [comment]
@silvau

You are correct, it is a library that needs to be included in the war file.  I will make a note to include it in the next war file build.

The notes on constructing your own war file: https://github.com/WillowIT/Pentaho-reports-for-OpenERP#building-and-installing

You will need to hunt through the files in those locations to find where it needs to be listed to ensure it's inclusion.  This is an area I am very sketchy on, so would need to spend time myself or need to ask around to work it out.

As an aside, the "ant" command may be better executed as "ant clean" and then "ant build".

Finally, let us know if it works, and either propose a merge or I can add it for the next build - that way we all can benefit ;)
